composer.json

```diff
@@ -26,14 +26,14 @@
         "nelmio/security-bundle": "~1.4",
         "doctrine/doctrine-migrations-bundle": "~1.0.0",
         "doctrine/migrations": "~1.0.0|1.0.0@alpha",
-        "mopa/bootstrap-bundle": "3.0.0-RC2",
+        "mopa/bootstrap-bundle": "3.0.0",
```

Comment:
Did you look into what changed between RC2 and stable and whether it affects us? Mopa Bootstrap has made many BC breaks during its journey to a stable 3.0.